### PR TITLE
Pcm 1738

### PR DESCRIPTION
**Pull request by djhouseknecht**

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -71,8 +71,8 @@ function validateOptions (options: ISdkConfig): string | null {
  * SDK to interact with GenesysCloud WebRTC functionality
  */
 export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEventEmitter<EventEmitter, SdkEvents> }) {
-  readonly VERSION = '__GENESYS_CLOUD_WEBRTC_SDK_VERSION__';
-  logger: ILogger;
+  static readonly VERSION = '__GENESYS_CLOUD_WEBRTC_SDK_VERSION__';
+  logger: Logger;
```
Comment (djhouseknecht) [inline src/client.ts:75]: This isn't really a breaking change, although I guess it _could_ mess with some typings potentially.

File: Jenkinsfile [hunk 2]
```diff
@@ -43,9 +43,8 @@ webappPipeline {
     postReleaseStep = {
         sshagent(credentials: [constants.credentials.github.inin_dev_evangelists]) {
             sh("""
-                # patch to prep for the next version
-                npm version patch --no-git-tag-version
-                git commit -am "Prep next version"
+                # patch to prep for the next version – this will create a commit
+                npm version patch -m "Prep next version: %s"
```
Comment (jensengar) [inline Jenkinsfile:47]: I don't think we want to do this because it will create a tag as well.

File: src/client-private.ts [hunk 3]
```diff
@@ -16,13 +17,24 @@ export async function setupStreamingClient (this: GenesysCloudWebrtcSdk): Promis
...
+    originAppVersion
+  } = this._config;
+
+  const connectionOptions: IClientOptions = {
     signalIceConnected: true,
-    host: this._config.wsHost || `wss://streaming.${this._config.environment}`,
-    apiHost: this._config.environment,
+    host: wsHost || `wss://streaming.${environment}`,
+    apiHost: environment,
     logger: this.logger,
```
Comment (jensengar) [inline src/client-private.ts:33]: We want to remove this logger now, right?

File: src/types/interfaces.ts [hunk 4]
```diff
@@ -115,6 +115,25 @@ export interface ISdkConfig {
...
 
+  /**
+   * This is name of the app that is consuming the SDK. This field is optional and only
+   *  used for loggering purposes.
+   */
+  originAppName?: string;
+
+  /**
+   * This is the version of the app that is consuming the SDK. This field is optional and only
+   *  used for loggering purposes.
```
Comment (zservies) [inline src/types/interfaces.ts:126]: loggering purposes 😂

**Review by zservies — APPROVED** [review]

File: src/types/interfaces.ts (hunk 4 above)
Comment (djhouseknecht) [reply src/types/interfaces.ts:126]: lol good catch. can you tell I was using my MBP builtin keyboard

**Review by maxwellmooney13 — APPROVED** [review]

**Pull request merged**